packages/kotti-ui/source/kotti-popover/KtPopoverIconTextItem.vue

```diff
@@ -0,0 +1,55 @@
+<template>
+	<div
+		v-bind="$attrs"
+		class="kt-popover-item"
+		:class="{
+			'kt-popover-item--disabled': isDisabled,
+		}"
+		v-on="$listeners"
```

Comment:
Same question

Reply:
v-on="$listeners" is used to catch the click event inside ktPopover